install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,205 @@
+= Mount Volumes on Privileged Pods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+== Overview
+
+The following provides a method for mounting persistent volumes to pods attached
+to the *privileged scc*.  For the purpose of this guide, GlusterFS will be the
+sample use-case though this method can be adapted to any
+link:../install_config/persistent_storage/index.html[supported storage configuration].
```

Comment:
I would suggest saying plugin rather than configuration. Configuration might be mis-leadsing because it entails more than just the storage plugin.